source/declarations.tex

```diff
@@ -2548,7 +2548,7 @@
 Otherwise, the terminal name of $E$ is not looked up.
 The declaration's target scope is the innermost enclosing namespace scope;
 if the declaration is contained by a block scope,
-the declaration shall correspond to a declaration
+the declaration shall correspond to a reachable\iref{module.reach} declaration
```

Comment:
Or just use English "an earlier", since we can't need to look into other translation units as for "precedes" or "reachable".

Reply:
Uh, but "earlier" also has a temporal meaning, which we don't want here.

Reply:
There is no time to apply here (although my initial mistake was more or less supposing that there was).

Reply:
Exactly, and that's why I don't want "earlier", which has a temporal aspect. In contrast, "reachable" has positional meaning (only), not temporal.